src/cowboy_session.erl

```diff
@@ -117,9 +117,10 @@ clear_cookie(Req) ->
 		Req2).
 
 create_session(Req) ->
-	%% The cookie value cannot contain any of the following characters:
-	%%   ,; \t\r\n\013\014
-	SID = list_to_binary(uuid:to_string(uuid:v4())),
+  SID = list_to_binary(uuid:to_string(uuid:v4())),
```

Comment:
if we're planning on contributing back, we should probably adhere to their _style guide_. Which seems to be 4 spaces?

Reply:
Oops, you're right. Good catch!